packages/components/forms/src/base-input/BaseInput.styles.ts

```diff
@@ -24,6 +23,11 @@ const getStyles = ({ isDisabled, isInvalid }) => ({
     cursor: isDisabled ? 'not-allowed' : 'auto',
     width: '100%',
 
+    // if the input is a textarea, the height is resizeble
```

Comment:
Can't this styling be passed from the `TextField` or `TextArea` component? Since emotion would merge the passed style with this, that should be the base style?

Reply:
but do you think it's correct if someone does this `<BaseInput as="textarea" />` and this textarea has a max-height of 40 px?

Reply:
Ah good point, I forgot that we would still export the `BaseInput`